### DISCUSSION TITLE
Shorthand pour acceptation de cookies spécifique ?

### DISCUSSION THREAD

**Discussion by PLefebvreBJO** [AmauriC/tarteaucitron.js#974] [Ideas]
Tout d'abord merci pour ce super projet c'est chouette de pouvoir respecter le RGPD sans devoir se taper des heures de code correspondant.

Je me demandais, est-ce qu'il est possible de recourir, au moyen d'un hashtag dans l'url par exemple, au switch d'un cookie spécifique ?

Dans mon cas, j'ai un portfolio qui s'affiche que si on accepte le cookie custom correspondant, sinon à la place on trouve une grosse div cliquable qui renvoie vers href #tarteaucitron pour paramétrer les cookies.
Je me demandais à quel point ça serait complexe d'implémenter le même système de hashtag juste pour accepter un cookie (en pratique ça donnerait un truc du genre "si vous souhaitez activer le cookie pour afficher ça, cliquez ici")

Je vais regarder un peu le code mais je pense que ça me dépasse alors je poste l'idée au cas où :)

![image](https://user-images.githubusercontent.com/99752136/181059286-42c060de-7e5e-4b49-8a60-7737bea43470.png)

**Comment by AmauriC** [comment]
Merci : D

C'est possible en javascript avec l'id du service (id de Google Maps dans l'exemple): 

```
tarteaucitron.userInterface.respond(document.getElementById('googlemapsAllowed'), true);
```

**Reply by PLefebvreBJO** [reply]
Oh super ! Ça marche à merveille merci beaucoup.
